packages/report-flaky-tests/lib/index.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import * as core from '@actions/core';
+
+/**
+ * Internal dependencies
+ */
+import { run } from './run';
+
+run().catch( ( error ) => {
+	core.error( error as Error );
```

Comment:
It probably doesn't matter in practice but a conditional is probably the safer way to coerce `error: any` into `error: Error | string`.

```ts
core.error( error instanceof Error ? error : error.toString() );
```